includes/class-wc-stripe.php

```diff
@@ -235,7 +235,7 @@ public function init() {
 			require_once WC_STRIPE_PLUGIN_PATH . '/includes/admin/class-wc-stripe-admin-notices.php';
 			require_once WC_STRIPE_PLUGIN_PATH . '/includes/admin/class-wc-stripe-settings-controller.php';
 
-			if ( isset( $_GET['area'] ) && 'payment_requests' === $_GET['area'] ) {
+			if ( isset( $_GET['area'] ) && 'express_checkout' === $_GET['area'] ) {
```

Comment:
This approach will break existing deep links to this page. I think we should either allow the old name, as per this suggestion
```suggestion
			if ( isset( $_GET['area'] ) && in_array( $_GET['area'], [ 'express_checkout', 'payment_requests' ], true ) ) {
```

I think it would be better if we redirect users to the new `area` when we receive the old one, though I am less sure of where that code should live. It may be simplest to accept both parameters in this PR for the time being, and handle the full redirect in a dedicated PR.
```php
			if (
				isset( $_GET['page'], $_GET['tab'], $_GET['section'], $_GET['panel'], $_GET['area'] ) &&
				'wc-settings' === $_GET['page'] &&
				'checkout' === $_GET['tab'] &&
				'stripe' === $_GET['section'] &&
				'methods' === $_GET['panel'] &&
				'payment' === $_GET['area']
			) {
				wp_safe_redirect( /* fixed URL */, WP_Http::MOVED_PERMANENTLY );
				exit;
			}
```

Reply:
Good idea 👍 . Suggestion applied